src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure createCircle() {
```

Comment:
Private ones (methods, classes) should be after public ones.
```suggestion
    private Figure createCircle() {
```